.readthedocs.yml

```diff
@@ -6,7 +6,7 @@ build:
     python: "3.11"
   jobs:
     pre_create_environment:
-      - git clone https://github.com/griptape-ai/griptape.git
+      - git clone -b dev https://github.com/griptape-ai/griptape.git
```

Comment:
@vasinov @andrewfrench this is horrible. Any suggestions for how to improve what value goes here?